jenkins/worker.py

```diff
@@ -0,0 +1,223 @@
+import argparse
```

Comment:
Run `pyformat` and `gpylint` and fix complains.

Reply:
Done.